napari/layers/image/image.py

```diff
@@ -415,7 +415,7 @@ def _calc_data_range(self, mode='data') -> Tuple[float, float]:
         if mode == 'data':
             input_data = self.data[-1] if self.multiscale else self.data
         elif mode == 'slice':
-            data = self._slice.image.view  # ugh
```

Comment:
This is probably the most important change to look at here. This should only matter for labels.

Reply:
I guessing because we pass the raw data through to vispy and because the data range isn't too relevant for labels, this is probably fine?

Reply:
+1 for `# ugh` removal!